Source/Scene/GroundPolylinePrimitive.js

```diff
@@ -382,7 +382,7 @@ define([
             return initPromise;
         }
 
-        GroundPolylinePrimitive._initPromise = ApproximateTerrainHeights.initialize('../Assets/approximateTerrainHeights.json')
+        GroundPolylinePrimitive._initPromise = ApproximateTerrainHeights.initialize()
```

Comment:
Looks like we won't need `_initializeTerrainHeightsWorker` anymore then. This is only getting called from `createGroundPolylineGeometry.js`.